Makefile

```diff
@@ -172,10 +171,9 @@ clean:
 distclean:
 	@rm -rf node_modules
 
-# create list of translations, saved as `./calypso-strings.php`
+# create list of translations, saved as `./calypso-strings.pot`
 translate: node_modules $(CLIENT_CONFIG_FILE)
-	@CALYPSO_ENV=stage $(BUNDLER)
-	@CALYPSO_ENV=stage $(LIST_ASSETS) | xargs $(I18N_CALYPSO) --format php --output-file ./calypso-strings.php --array-name calypso_i18n_strings
+	@CALYPSO_ENV=stage $(I18N_CALYPSO) --format pot --output-file ./calypso-strings.pot $(JS_FILES)
```

Comment:
The assets files are used for looking up bundled script locations by chunk, so they'll need to remain. That being said, there's some change in behavior here then, as previously `I18N_CALYPSO` was run on the built scripts, and are now being run on the source files. Is that okay?